app_legacy/Helpers/render/game.php

```diff
@@ -84,7 +94,22 @@ function renderGameTitle(?string $title = null, bool $tags = true): string
         $updateHtml($html, $matches[0], $tags ? " $span" : '');
     }
 
-    return $html;
+    return "<div>$html</div>";
+}
+
+/**
+ * Render console icon based on given system ID
+ *
+ * Fallback to a default image if icon fails to be found on server
+ */
+function renderConsoleIcon(string $consoleName, int $consoleID): string {
+    $fallBackConsoleIcon = asset("assets/images/system/unknown.png");
+    $cleanSystemShortName = Str::lower(str_replace("/", "", config("systems.$consoleID.name_short")));
+    $iconName = Str::kebab($cleanSystemShortName);
+    $iconPath = public_path("assets/images/system/$iconName.png");
+    $iconUrl = file_exists($iconPath) ? asset("assets/images/system/$iconName.png") : $fallBackConsoleIcon;
+
+    return "<img class='console-icon' src='$iconUrl' alt='$consoleName icon'>";
 }
```

Comment:
Suggestion: Let's turn this into a Blade component and not add another render function to legacy. resources/views/components might be a good fit as this is fairly generic and could exist on any page throughout the site:

```
<x-console-icon consoleName="Mega Drive" consoleId="1" />
```

It seems everywhere this function is being used, it's being `echo`'d out. A swap to using `Blade::render()` should work nicely and prevent us from having to migrate this later.